.github/workflows/pypi_release.yml

```diff
@@ -0,0 +1,43 @@
+name: PyPI Releases
+
+on:
```

Comment:
Should it be done only when any of the related jobs is done and sucessfully done ?